sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/Channel.java

```diff
@@ -48,16 +48,18 @@
     /**
      * Pauses the given group.
      *
-     * @param groupName the name of a group.
+     * @param groupName   the name of a group.
+     * @param targetToken the target token to pause, or null to pause the entire group.
```

Comment:
Does this mean that `pause(groupName, null)` on a transmission target has the same behavior as `pause()` on analytics?  I'm not sure about the difference.

Reply:
Analytics.pause() calls pauseGroup(group, null)
AnalyticsTransmissionTarget.pause() calls pauseGroup(group, token)